airbyte-cdk/bulk/toolkits/extract-jdbc/src/testFixtures/resources/application.yml

```diff
@@ -1,6 +1,20 @@
 ---
 airbyte:
   connector:
+    check:
+      jdbc:
+        queries:
+          - SELECT DATABASE_PATH() FROM DUAL
```

Comment:
this returns NULL For in-memory H2 database instances, which is what we use in our tests